### PR TITLE
fix(replays): Disable broken test

### PR DESCRIPTION
**Pull request by cmanallen**

### CODE REVIEW

File: tests/sentry/replays/test_organization_replay_index.py [hunk 1]
```diff
@@ -1249,81 +1249,81 @@ def test_get_replays_memory_error(self):
                     == b'{"detail":"Replay search query limits exceeded. Please narrow the time-range."}'
                 )
 
-    def test_get_replays_dead_rage_click_cutoff(self):
```
Comment (asottile-sentry) [inline tests/sentry/replays/test_organization_replay_index.py:1252]: please follow the procedure from https://github.com/getsentry/rfcs/blob/main/text/0027-manual-disabling-of-flaky-tests.md

Comment (cmanallen) [reply tests/sentry/replays/test_organization_replay_index.py:1252]: @asottile-sentry done

**Review by asottile-sentry — APPROVED** [review]
ideally there would be a ticket associated with fixing it too

**Comment by cmanallen** [comment]
@asottile-sentry Handled :D https://github.com/getsentry/team-replay/issues/150

**Pull request merged**